taxi/views.py

```diff
@@ -34,6 +35,26 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
According to the checklist, you should not override `fields` if they are the same as the default ones. Consider specifying only the fields that need to be customized.